lib/delete.js

```diff
@@ -1,17 +1,17 @@
-const AWS = require('aws-sdk');
+const { S3Client, DeleteObjectCommand, S3ServiceException} = require('@aws-sdk/client-s3');
 const awsError = require('./error.js');
 const parallel = require('parallel-stream');
 
 module.exports = function(bucket, options) {
   options = options || {};
 
   const s3config = {
-    maxRetries: 10,
-    httpOptions: { connectTimeout: 3000 }
+    maxAttempts: 10,
```

Comment:
would we want to make this 11 to account for the base attempt? (the maxRetries is 10 above)